internal/pkg/panic/panic.go

```diff
@@ -0,0 +1,69 @@
+package panic
+
+import (
+	"runtime"
+	"runtime/debug"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"golang.org/x/exp/slices"
+
+	cliv1 "github.com/confluentinc/ccloud-sdk-go-v2/cli/v1"
+
+	v1 "github.com/confluentinc/cli/internal/pkg/config/v1"
+	"github.com/confluentinc/cli/internal/pkg/usage"
+)
+
+// CollectPanic collects relevant usage data for when panics occur and command execution is not completed.
+func CollectPanic(cmd *cobra.Command, args []string, cfg *v1.Config) *usage.Usage {
+	fullCommand, flags, _ := cmd.Find(args)
+	return &usage.Usage{
+		Os:          cliv1.PtrString(runtime.GOOS),
+		Arch:        cliv1.PtrString(runtime.GOARCH),
+		Version:     cliv1.PtrString(cfg.Version.Version),
+		Command:     cliv1.PtrString(fullCommand.CommandPath()),
+		Flags:       parseFlags(fullCommand, flags),
+		Error:       cliv1.PtrBool(true),
+		StackFrames: parseStack(string(debug.Stack())),
+	}
+}
+
+// parseFlags collects the flags alongside the panicking command
+func parseFlags(cmd *cobra.Command, flags []string) *[]string {
+	var formattedFlags []string
+	for i := range flags {
+		flags[i] = strings.TrimPrefix(flags[i], "--")
+		flags[i] = strings.TrimPrefix(flags[i], "-")
```

Comment:
```suggestion
		flags[i] = strings.TrimLeft(flags[i], "-")
```
TrimLeft will remove any number of `-`'s so we don't need to code for both cases.